src/app/seamly2d/mainwindow.cpp

```diff
@@ -53,298 +53,358 @@
+        initializeDocksContain();       
+        createMenus();
 
-    initializeToolsToolBar();
+        // Initialize toolbars for draft, point name, and modes
+        initializeDraftToolBar();
+        initializePointNameToolBar();
+        initializeModesToolBar();
 
-    connect(qApp->getUndoStack(), &QUndoStack::cleanChanged, this, &MainWindow::patternChangesWereSaved);
+        // Initialize tool buttons and show the main window maximized
+        initializeToolButtons();
+        showMaximized();
 
-    initializeAutoSave();
+        initPenToolBar();
 
-    ui->draft_ToolBox->setCurrentIndex(0);
+        // Create a help label and add it to the status bar
+        helpLabel = new QLabel(QObject::tr("Create new pattern piece to start working."));
+        ui->statusBar->addWidget(helpLabel);
 
-    ReadSettings();
-    initializeToolBarVisibility();
+        initializeToolsToolBar();
 
-    setCurrentFile("");
-    WindowsLocale();
+        // Connect the undo stack cleanChanged signal to the patternChangesWereSaved slot
+        connect(qApp->getUndoStack(), &QUndoStack::cleanChanged, this, &MainWindow::patternChangesWereSaved);
 
-    connect(ui->listWidget, &QListWidget::currentRowChanged, this, &MainWindow::showLayoutPages);
+        initializeAutoSave();                       // Initialize pattern auto-save functionality
+        ui->draft_ToolBox->setCurrentIndex(0);      // Set the current index for the draft toolbox
 
-    connect(watcher, &QFileSystemWatcher::fileChanged, this, &MainWindow::MeasurementsChanged);
-    connect(qApp, &QApplication::focusChanged, this, [this](QWidget *old, QWidget *now)
-    {
-        if (old == nullptr && isAncestorOf(now) == true)
-        {// focus IN
-            static bool asking = false;
-            if (!asking && mChanges && not mChangesAsked)
-            {
-                asking = true;
-                mChangesAsked = true;
-                const auto answer = QMessageBox::question(this, tr("Measurements"),
-                                                 tr("Measurements were changed. Do you want to sync measurements now?"),
-                                                          QMessageBox::Yes|QMessageBox::No, QMessageBox::Yes);
-                if (answer == QMessageBox::Yes)
+        // Read application settings and initialize toolbar visibility
+        ReadSettings();
+        initializeToolBarVisibility();
+
+        // Set the current file to an empty string and apply Windows locale settings
+        setCurrentFile("");
+        WindowsLocale();
+
+        connect(ui->listWidget, &QListWidget::currentRowChanged, this, &MainWindow::showLayoutPages);   // Connect the list widget's currentRowChanged signal to the showLayoutPages slot  
+        connect(watcher, &QFileSystemWatcher::fileChanged, this, &MainWindow::MeasurementsChanged);     // Connect file system watcher to handle measurement changes      
```

Comment:
Same, and there it makes the line very long, which displays part of the comment on the next line in some IDE, which makes the whole thing more difficult to read